### DISCUSSION TITLE
Move语言配置服务器&搭建节点测试业务

### DISCUSSION THREAD

**Discussion by ivan9230** [NonceGeek/Move-Camp#145] [Q&A] [answered]
Hi all,
问题1 本人现在用Macair 2015，请问这个电脑本学习move语言可否?如果不行的话最性价比的方案是租一个阿里云服务器，问题选哪个级别和价格? 另外我想把这个服务器同时搞节点测试，可否可以? 

问题2 就是怎么批量账号搭建搞节点测试aptos和sui? sui是否也同样看好能获益?是否有相关教程攻略?有没有这类群推荐小伙伴可以一起搞呢？

Best Regards
谢谢老师

**Comment by leeduckgo** [comment, marked as answer]
1. 你的 Mac 可以；
2. 可以通过租用境外服务器来做，可以看好 Sui 的收益，在群里组队即可。
